addons/l10n_id/models/account_move.py

```diff
@@ -1,25 +1,11 @@
 # Part of Odoo. See LICENSE file for full copyright and licensing details.
-import pytz
-
 from odoo import fields, models, _
 
 
 class AccountMove(models.Model):
     _inherit = 'account.move'
 
-    # Used to fetch the QR payment status.
-    # List of dicts with the format: [{
-    #     'qris_invoice_id': '13246',
-    #     'qris_amount': 0.0,
-    #     'qris_creation_datetime': '2024-02-27 03:00:00',
-    #     'qris_content': 'xxx',
-    # }]
-    l10n_id_qris_invoice_details = fields.Json(
-        string="QRIS Transaction Number",
-        help="Transaction Number stored.",
-        readonly=True,
-        export_string_translation=False,
-    )
+    l10n_id_qris_transaction_ids = fields.Many2many('l10n_id.qris.transaction')
```

Comment:
This needs an upgrade script:
1. A field was removed
2. The data must be moved to a table

There may be more issues. Please do this asap @nni-odoo @vin-odoo

Reply:
Hello Alvaro, sorry about that.
We had planned not to migrate these data as they represent unfulfilled transactions that are only valid 30m and are most likely no longer relevant by the time the migration has finished.
We did run the upgrade tests but as it was pointed out on discord they don't really work for l10n :sweat_smile:
But we should have been more careful as we remove a field

I see that Christophe has a commit to migrate these data. Do you still need us to do it?
I noticed that this commit has a small issue (bank_id must be filled with the invoice partner_bank_id)

Reply:
Hi @vin-odoo

> We had planned not to migrate these data as they represent unfulfilled transactions that are only valid 30m and are most likely no longer relevant by the time the migration has finished.

You are the expert so please review the linked commit :) Take into account that RR upgrades may happen in a matter of a few minutes (<10) thus the data could still be valid. Unless some other functional change invalidates it I think we should move it, up to you to decide.

> We did run the upgrade tests but as it was pointed out on discord they don't really work for l10n 😅
> But we should have been more careful as we remove a field

Yes, wee need to ensure runbot tests this.